<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerDashboard</class>
 <widget class="QWidget" name="ServerDashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>837</width>
    <height>866</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>List</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableFile">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>621</width>
     <height>261</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>File Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Username</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Upload Date</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>File Size</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Control</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonDeleteFile">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>370</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSearchFile">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>320</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editSearchFile">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>320</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableUser">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>440</y>
     <width>621</width>
     <height>261</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Username</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Password</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="buttonDeleteUser">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>770</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>430</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>List</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>720</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Control</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonRefreshFile">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>370</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonRefreshUser">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>770</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
